.github/dependabot.yml

```diff
@@ -75,7 +77,8 @@ updates:
     open-pull-requests-limit: 20
```

Comment:
Bump this up.
We have in 11.0.x an average of 4.2 dependabot PRs per week.
Lets make 11.0.x have a limit above 20, lets use 30.

Raw numbers from the start of the year.
```
11.0.x - Weekly updates
 
  is:pr created:>=2022-01-01 head:jetty-11.0.x
      2 open
     98 closed
   (4.4 PRs per week)
  is:pr created:>=2022-01-01 base:jetty-11.0.x label:dependencies 
      2 open
     93 closed
   (4.2 PRs per week)
  is:pr created:>=2022-01-01 base:jetty-11.0.x label:dependencies is:merged 
     56 merged
   (2.5 PRs per week)
  is:pr created:>=2022-01-01 base:jetty-11.0.x label:dependencies is:unmerged 
     39 un-merged
   (70% not merged)
```